MdePkg/Library/BasePanicLibNull/BasePanicLibNull.uni

```diff
@@ -0,0 +1,16 @@
+// /** @file
+// Panic Library with empty functions.
+//
+// Panic Library with empty functions.

```

Comment:
```suggestion
```

Reply:
I removed the file with my latest commit.